src/lib/apiClient.ts

```diff
@@ -0,0 +1,23 @@
+const API_URL = 'http://localhost:3000/api';
+
+export const apiClient = {
+  publishMyProfile: async (profileId: string) => {
+    if (!profileId) {
+      throw new Error('No profile ID found');
```

Comment:
**nitpick:** You don't have to check it if passed argument is not optional? This `if` is useless here